packages/easy-coding-standard/build/target-repository/.github/workflows/test.yaml

```diff
@@ -0,0 +1,23 @@
+name: Fixer Test
```

Comment:
Better file name than `test` would be better :D

Reply:
phpunit then? 🤔

Reply:
What is this testing? PHPUnit?